mlflow/tracking/client.py

```diff
@@ -592,9 +592,8 @@ def start_trace(
 
             return mlflow_span
         except Exception as e:
-            _logger.warning(f"Failed to start span {name}: {e}")
-            if _MLFLOW_TESTING.get():
-                raise
+            _logger.warning(f"Failed to start trace {name}: {e}")
+            _logger.debug("Showing full traceback: ", exc_info=True)
```

Comment:
```suggestion
            _logger.warning(f"Failed to start trace {name}: {e}. For full traceback, set logging level to debug.")
            _logger.debug("Showing full traceback: ", exc_info=True)
```

I'd include `For full traceback, set logging level to debug.`, otherwise no one will notice.

Reply:
not a blocker, can be a follow-up